backend/data/events.model.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require("mongoose");
+
+const UserSchema = mongoose.Schema({
+  _id: mongoose.Schema.Types.ObjectId,
```

Comment:
@MHDGhanemBalhawan We using mongoose with postgress ?

Reply:
It looks like we don't have a dependency on mongoose, so I think this file can be removed.